.do/values.yaml

```diff
@@ -174,3 +176,32 @@ crow:
         context: "/monitoring"
     passwordAuth: {}
   crowdb: {}
+
+rabbitmq-ha:
+  metrics:
+    enabled: true
+    serviceMonitor:
+      enabled: true
+    prometheusRule:
+      enabled: true
+      rules:
+        - alert: Rabbitmq Unavailable
+          expr: |
+            kube_statefulset_status_replicas_ready{statefulset="{{ template "rabbitmq.fullname" . }}"} < kube_statefulset_status_replicas{statefulset="{{ template "rabbitmq.fullname" . }}"}
+          for: 15m
+          labels:
+            severity: warning
+          annotations:
+            summary: Insufficient Rabbitmq Nodes
+            description: |
+              Insufficient Rabbitmq Nodes - {{ "{{ $value }}" }}
+        - alert: Rabbitmq Down
+          expr: |
+            kube_statefulset_status_replicas{statefulset="{{ template "rabbitmq.fullname" . }}"} == 0
+          for: 5m
+          labels:
+            severity: error
+          annotations:
+            summary: Rabbitmq Down
+            description: |
+              No Rabbitmq Nodes
```

Comment:
This should alert us when the entire rabbitmq cluster has issues starting all the replicas.